doc/api/crypto.md

```diff
@@ -249,11 +249,11 @@ added: v1.0.0
   - `plaintextLength` {number}
 * Returns: {Cipher} for method chaining.
 
-When using an authenticated encryption mode (`GCM`, `CCM` and `OCB` are
+When using an authenticated encryption mode (`GCM`, `CCM`, `OCB` and `AEAD_CHACHA20_POLY1305` are
```

Comment:
Please wrap lines at a width of 80 characters.

Reply:
Sure, I will fix this